### PR TITLE
Cryptocurrency tutorial 0.10

### PR DESCRIPTION
**Pull request by Maria-Nosyk**

### CODE REVIEW

File: src/get-started/create-service.md [hunk 1]
```diff
@@ -86,42 +93,114 @@ const INIT_BALANCE: u64 = 100;
...
-encoding_struct! {
-    struct Wallet {
-        pub_key: &PublicKey,
-        name: &str,
-        balance: u64,
-    }
+#![allow(bare_trait_objects)]
+#![allow(renamed_and_removed_lints)]
+
+pub use self::cryptocurrency::Wallet;
```
Comment (ivan-ochc) [inline src/get-started/create-service.md]: Maybe It makes sense to use all structs from the beginning, including: TxCreateWallet, TxTransfer?

File: src/get-started/create-service.md [hunk 2]
```diff
@@ -619,27 +722,29 @@ in the Exonum network.
...
 ```
 
 ### Run Node
 
-Finally, we need to implement the entry point to our demo network – `main` function:
+Finally, we need to implement the entry point to our demo network – `main`
+function:
 
 ```rust
 fn main() {
```
Comment (ivan-ochc) [inline src/get-started/create-service.md]: MemoryDB? Seems that this part is outdated.

Comment (ivan-ochc) [inline src/get-started/create-service.md]: Argument has been missed in constructor. Actual is:
```
let node = Node::new(
        MemoryDB::new(),
        vec![Box::new(CurrencyService)],
        node_config(),
        None,
    );
```

File: src/get-started/create-service.md [hunk 3]
```diff
@@ -499,54 +597,59 @@ we should implement the `Service` trait in it.
...
-this API instance to the `ServiceApiBuilder`.
-
 ## Create Demo Blockchain
 
 The service is ready. You can verify that the library code compiles by running
-`cargo build` in the shell. However, we do not have the means of processing requests
+`cargo build` in the shell. However, we do not have the means of processing
+requests
 to the service. To fix this, let us create a minimalistic blockchain network
 with one node and a single service we’ve just finished creating.
```
Comment (ivan-ochc) [inline src/get-started/create-service.md]: Shouldn't we provide list of required imports for `examples/demo.rs` ?

File: src/get-started/create-service.md [hunk 4]
```diff
@@ -619,27 +722,29 @@ in the Exonum network.
 ```rust
 let peer_address = "0.0.0.0:2000".parse().unwrap();
```
Comment (ivan-ochc) [inline src/get-started/create-service.md]:
```suggestion
let peer_address = "0.0.0.0:2000";
```

**Comment by dvush** [comment]
@ivan-ochc I fixed code here https://github.com/Maria-Nosyk/exonum-doc/pull/2
I checked by copy-pasting everything into new project and it seems to work fine.

**Pull request merged**